src/main/java/frc/utils/ToleranceUtils.java

```diff
@@ -0,0 +1,21 @@
+package frc.utils;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.constants.MathConstants;
+
+public class ToleranceUtils {
+
+	public static boolean wrappedIsNear(Rotation2d tolerance, Rotation2d rotation, Rotation2d wantedRotation) {
+		return Math.abs(wantedRotation.getRadians() - rotation.getRadians()) <= tolerance.getRadians()
+			|| Math.abs(wantedRotation.getRadians() - rotation.getRadians()) >= MathConstants.FULL_CIRCLE.getRadians() - tolerance.getRadians();
```

Comment:
doesn't this not work if the numbers are like: 0, 600? you need to do modulus

Reply:
I am quite sure the Rotation2d does the modulu